fxgl-ai/src/test/java/com/almasb/fxgl/pathfinding/astar/AStarPathfinderTest.java

```diff
@@ -80,8 +79,9 @@ public void testFindPathWithBusyCells() {
                 3, 4,
                 4, 4,
                 4, 5);
-        path = pathfinder.findPath(1, 1, 4, 5, Collections.singletonList(grid.get(3, 4)));
-        assertPathEquals(path,
+        List<AStarCell> pathWithBusyCell = pathfinder.findPath(1, 1, 4, 5, Collections.singletonList(grid.get(3, 4)));
+        System.out.println("Solution: " + pathWithBusyCell);
```

Comment:
Could we revert changes to the test file. I don't think you meant to commit these.

Reply:
I actually meant to, because I prefer to have meaningful output in the test so you actually now what the error is. Because if the error says: 2 not equal 1, then I will not know anything. 

But of course I can revert this if it doesn't fit your approach.

Reply:
Good point about output. Adding `System.out` here would be inconsistent compared to other tests. So I think the test can be improved, particularly the bit that compares paths. Can we rewrite that, so that `assertThat` correctly prints the output when failing? Something like `assertThat(list1, is(list2))`?